src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/VariantArray16FW.java

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright 2016-2019 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.reaktivity.nukleus.maven.plugin.internal.generated;
+
+import java.util.function.Consumer;
+
+import org.agrona.BitUtil;
+import org.agrona.DirectBuffer;
+import org.agrona.MutableDirectBuffer;
+import org.agrona.concurrent.UnsafeBuffer;
+import org.reaktivity.reaktor.internal.test.types.Flyweight;
+
+public final class VariantArray16FW<V extends VariantFW<?, ? extends Flyweight>> extends VariantArrayFW<V>
```

Comment:
I don't think you need the `extends Flyweight` part when using `?` since it is not bound to a type variable that needs to be verified in other places.